tailwind.config.js

```diff
@@ -110,6 +110,11 @@ module.exports = {
 
       blue: {
         200: '#0082FA'
+      },
+
+      orange: {
+        500: '#F3610F',
+        '500-faded': 'rgba(243, 97, 15, 0.1)'
```

Comment:
Can we do this instead `bg-orange-500 bg-opacity-10`, so not to mix opacity with color declaration in tailwind config.

Reply:
Sure, should I also do the same for line 61?